ramp-database/ramp_database/tools/submission.py

```diff
@@ -32,6 +32,7 @@
 from ._query import select_submission_file_type_by_name
 from ._query import select_team_by_name
 
+CACHE_DIR = '/home/ramp/ramp_deployment/cache/'
```

Comment:
I think this should not be hardcoded here, rather handled using config.

Reply:
We used sqlalchemy to differ the loading. We don't need caching anymore.